redisson/src/main/java/org/redisson/misc/ProxyBuilder.java

```diff
@@ -50,6 +50,7 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
                 Method instanceMethod = getMethod(method, instance, implementation);
 
                 if (instanceMethod.getName().endsWith("Async")) {
+                    instanceMethod.setAccessible(true);
```

Comment:
Why do we need this?

Reply:
Without this, several method invocations fail in `RedissonReactiveListMultimapTest` eventhough the target methods are marked public:

> Caused by: java.lang.IllegalAccessException: class org.redisson.misc.ProxyBuilder$1 cannot access a member of class org.redisson.RedissonListMultimap$1 with modifiers "public"
	at org.redisson.misc.ProxyBuilder$1.lambda$invoke$0(ProxyBuilder.java:54)

> removeAll(Collection)
> addAll(Collection)
> addAll(int, Collection)
> delete()
> removeAll(Collection)

Reply:
Fixed in https://github.com/redisson/redisson/commit/929bfb4616bea49eb9028766fa068a1fa3974cb6